.storybook/stories/CheckBox/CheckBox.tsx

```diff
@@ -26,8 +28,11 @@ export const CheckBox: FC<CheckBoxProps> = ({
         onPress={onPress}
         color={color}
         uncheckedColor={color}
+        disabled={disabled}
       />
-      {label && <Typography text={label} variant="body-medium" />}
+      {label && (
+        <Typography text={label} variant="body-medium" color="text-secondary" />
```

Comment:
I don't think setting labels as text-secondary is a good idea, it's providing crucial information, so it should be primary